modules/monitoring-configuring-cluster-for-application-monitoring.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// monitoring/monitoring.adoc
+
+[id='configuring-cluster-for-application-monitoring-{context}']
+= Configuring cluster for application monitoring
+
+Before application developers can monitor their applications, the human operator of the cluster needs to configure the cluster accordingly. This procedure shows how to.
+
+.Prerequisites
+
+* You need to log in as a user belonging to a role with administrative privileges for the cluster.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Catalog* -> *Operator Hub* and install the Prometheus Operator.
```

Comment:
It is "Catalog → OperatorHub" in web console and should add "on the namespace you choose" in the end